site/en/workflows.md

```diff
@@ -253,6 +295,20 @@ For better results, the cropped images can be binarized again at this point or l
       </td>
       <td>Recommended</td>
       <td><code>ocrd-olena-binarize -I OCR-D-CROP -O OCR-D-BIN2 -p'{"impl": "sauvola-ms-split"}'</code></td>
+    </tr>
+	<tr>
+      <td>ocrd-skimage-binarize</td>
+      <td>
+	  </td>
+      <td>
+	  </td>
+      <td><code>ocrd-skimage-binarize -I OCR-D-IMG -O OCR-D-BIN</code></td>
```

Comment:
why do I need the parameter in this case? Isn't sauvola the default value anyway?

Reply:
It is. I thought the usage of parameters here was more for illustrative purposes than actual necessities (which are hard to get right anyway)...

Reply:
ok, I see. At least I try just to use parameters which are actually required by a processor. Otherwise it would get even more confusing and complex. And we wanted this workflow page to just give a relatively easy and short overwiew for OCR-D-beginners...